src/magicArtists.js

```diff
@@ -0,0 +1,433 @@
+const artists = [
```

Comment:
[reccomendation] Is this list of artists from the API? If so it might be worth working in a second fetch call to populate the lists of artist on page load?